testnets/xiontestnet/assetlist.json

```diff
@@ -181,6 +174,52 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdc.svg"
       }
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/0166AE2224341A3F70943E315DAC6EDF012A638D0F9358794FF7525BA1DFC191",
+          "exponent": 0,
+          "aliases": [
+            "inj"
+          ]
+        },
+        {
+          "denom": "INJ",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/0166AE2224341A3F70943E315DAC6EDF012A638D0F9358794FF7525BA1DFC191",
+      "name": "Injective INJ token",
+      "display": "inj",
+      "symbol": "INJ",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injectivetestnet",
+            "base_denom": "inj",
+            "channel_id": "channel-324"
```

Comment:
Please register this IBC connection

Reply:
make sense, thank you. updated to `INJ`

Reply:
registered to `_IBC`, mirroring live connection. thank you

```
# Chain: injective-888
  - Client: 07-tendermint-228
    * Connection: connection-211
      | State: OPEN
      | Counterparty state: OPEN
      + Channel: channel-324
        | Port: transfer
        | State: OPEN
        | Counterparty: channel-215
# Chain: xion-testnet-1
  - Client: 07-tendermint-105
    * Connection: connection-43
      | State: OPEN
      | Counterparty state: OPEN
      + Channel: channel-215
        | Port: transfer
        | State: OPEN
        | Counterparty: channel-324
```